packages/components/src/Icons/AcademyIcon.tsx

```diff
@@ -0,0 +1,19 @@
+// This file is autogenerated by wrapSVGs.ts
+// Changes to this file will be overwritten
+
+import React from "react"
+import { v4 as uuidv4 } from "uuid"
+import { SVG, SVGProps } from "~components/Icons/subComponents/SVG"
+
+const uniqueId = uuidv4()
```

Comment:
I think this is the cause for your a11y errors since this might not actually run every time the icon is rendered, try placing this inside the component's function.